src/app/(route)/all-link/AllLink.tsx

```
@@ -96,6 +142,22 @@ export default function AllLink() {
       </div>
+
+      {type === 'DELETE_LINK' && (
+        <DeleteLinkModal
+          links={links
+            .filter(l => props.linkIds.includes(l.id))
+            .map(l => ({ id: l.id, title: l.title, url: l.url }))}
+          onSuccess={succeededIds => {
+            // 삭제 성공한 id만 선택 해제
+            setSelectedIds(prev => {
+              const next = new Set(prev);
+              succeededIds.forEach(id => next.delete(id));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)